db/schema.rb

```diff
@@ -275,6 +276,28 @@
     t.index ["phone_number"], name: "index_channel_whatsapp_on_phone_number", unique: true
   end
 
+  create_table "companies", force: :cascade do |t|
```

Comment:
😄  might be a good idea to delete the companies table from your local DB unless it is being used.